pkg/network/protocols/http2/protocol.go

```diff
@@ -119,8 +127,10 @@ func newHTTP2Protocol(cfg *config.Config) (protocols.Protocol, error) {
 	telemetry := http.NewTelemetry("http2")
 
 	return &protocol{
-		cfg:       cfg,
-		telemetry: telemetry,
+		cfg:                     cfg,
+		telemetry:               telemetry,
+		terminatedConnections:   make([]netebpf.ConnTuple, 0),
```

Comment:
A `nil` slice is treated the same as an empty slice, so no need for this `make` call.